paper-toast.html

```diff
@@ -187,6 +187,8 @@
          * Called when the value of `opened` changes.
          */
         _openedChanged: function() {
+          this.cancelAsync(this._autoClose);
+          this._autoClose = undefined;
```

Comment:
also, `cancelAsync` doesn't like `null` as input, so it would require an extra-check :x